### PR TITLE
Porting to TypeScript

### PR DESCRIPTION
**Pull request by delaaxe**
Port the config loader, the multisig executor and an example deployment script to TypeScript

### CODE REVIEW

File: scripts/utils/multisigexecutor.ts [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-  constructor(autoSign = true, setupGas = false) {
-    this._setupGas = setupGas;
-    this._autoSign = autoSign;
-  }
+  constructor(private _autoSign = true, private _setupGas = false) {}
 
-  async connect(multisigAddress) {
-    this._deployer = await ethers.getSigner();
+  async connect(multisigAddress: string) {
+    [this._deployer] = await ethers.getSigners();
```
Comment (janek26) [inline scripts/utils/multisigexecutor.ts:14]: does this work? 🤯 nice one

File: scripts/utils/multisigexecutor.ts [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
-const inquirer = require("inquirer");
+import { Contract } from "@ethersproject/contracts";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { ethers } from "hardhat";
+import inquirer from "inquirer";
+import multisigAbi from "./multisig.json";
 
-const multisigAbi = require('./multisig.json');
+export class MultisigExecutor {
+  private _deployer!: SignerWithAddress;
```
Comment (janek26) [inline scripts/utils/multisigexecutor.ts:8]: `!` shouldn't be needed. If you don't specify it as `?` TS will require you to assign it in the constructor

**Review by janek26 — APPROVED** [review]

File: scripts/utils/multisigexecutor.ts (hunk 2 above)
Comment (delaaxe) [reply scripts/utils/multisigexecutor.ts:8]: The thing is that I didn't want to change the constructor that's use in 10+ other files in this first smaller PR (the common usage pattern is [like this](https://github.com/argentlabs/argent-trustlists/blob/develop/scripts/deploy-aave.js#L23))

File: scripts/utils/multisigexecutor.ts (hunk 1 above)
Comment (delaaxe) [reply scripts/utils/multisigexecutor.ts:14]: moreover `getSigner` isn't meant to be used like this, even though it works 😅

**Pull request merged**